spec/oncocore_genetics.txt

```diff
@@ -27,13 +31,13 @@ Description:	"Results of analyzing a specified genetic region to detect the pres
 				DataValue from DetectedNotDetectedVS  // mlt:  changed to mk-preferred value set which replaces PresentAbsentVS with DetectedNotDetectedVS.  For CGWG, this ideally should be constrained to present|absent|indeterminate.
 0..0			BodyLocation
 				Components.ObservationComponent
-					includes 0..*	GeneName
+					includes 0..*	GeneName // GeneStudied
```

Comment:
okay, after quickly looking through your latest PR, I see what you mean re: `GeneName`.  It should never have been that.  I think I originally called it `GeneStudied` and it was then changed to `GeneName` (I don't remember who changed it).  Regardless, if we keep it to the same elementName `GeneStudied` (which is what the LOINC code description for the bound code actually is), then the rest of the fields related to Gene ID, and Gene Symbol below it make more sense.

Reply:
I'll make a change and we'll re-evaluate.